.gitlab-ci.yml

```diff
@@ -5,30 +5,17 @@ before_script:
   - cp /systems/osc_certs/gpg/ondemand/.gpgpass $CI_PROJECT_DIR/tmp/ondemand-packaging/
   - cp /systems/osc_certs/gpg/ondemand/ondemand.sec $CI_PROJECT_DIR/tmp/ondemand-packaging/
 stages:
-  - build
-  - deploy
+  - build_and_deploy
 
-rpm-build:
-  stage: build
-  only:
-    - tags
-  script:
-    - ./tmp/ondemand-packaging/build.sh -w $CI_PROJECT_DIR/tmp/work -o $CI_PROJECT_DIR/tmp/output -V $CI_COMMIT_TAG -u $CI_PROJECT_DIR/packaging
-  artifacts:
-    paths:
-      - tmp/output
-    name: "$CI_PROJECT_NAME-$CI_COMMIT_TAG"
-
-
-rpm-deploy:
+rpm-build_and_deploy:
```

Comment:
This should be `build_and_deploy` or change name under `stages` to match this value.

Reply:
Actually nevermind, forgot stage is defined further down.